docs/ref/forms/widgets.txt

```diff
@@ -577,6 +577,23 @@ These widgets make use of the HTML elements ``input`` and ``textarea``.
     format found in :setting:`DATE_INPUT_FORMATS` and respects
     :doc:`/topics/i18n/formatting`.
 
+.. admonition:: Date picker implementation
+
+    By default, the :class:`~django.forms.DateInput` is rendered as a simple
+    text box, i.e. ``<input type="text" ...>``. If you prefer a full-fledged
+    date picker instead, you can take advantage of the browser's interface for
+    the `HTML5 "date" input type`_. This can be achieved by overriding the
+    widget's ``input_type`` attribute, for example: ::
+
+        class MyDateInput(forms.DateInput):
+            input_type = 'date'
+
+    Alternatively, you could specify the widget as follows: ::
+
+        DateInput(attrs={'type': 'date'})
+
+.. _HTML5 "date" input type: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date
+
```

Comment:
I don't think we need the full `admonition` call out here. 

Just paragraphs should suffice:


```suggestion
    By default, :class:`~django.forms.DateInput` is rendered as an ``input``
    with ``type=text``. If you are targetting browsers which support the the
    `HTML5 "date" input type`_, you may override the ``input_type`` attribute,
    for example::
    
        class MyDateInput(forms.DateInput):
            input_type = 'date'

    Alternatively, you can specify the widget `attrs` directly::

        DateInput(attrs={'type': 'date'})

.. _HTML5 "date" input type: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date        
```

Reply:
Thanks @carltongibson, that does look cleaner. Good point mentioning browser support. Personally I would prefer to mention "date picker" explicitly, somewhere in there: I expect quite a few users will be unaware of the fact that most browsers implement a fully-functional date picker interface for inputs with `type="date"`. At least it took *me* a while to find that out. ;-)

Reply:
@dennisvang — OK, have a stab — it's a _suggestion_. 😀